CosmosApiChangeFeedForMessageRetention/function.json

```
@@ -0,0 +1,17 @@
+{
+  "bindings": [
+    {
+      "type": "cosmosDBTrigger",
+      "name": "documents",
+      "direction": "in",
+      "leaseCollectionName": "%LEASE_MESSAGE_STATUS%",
+      "leaseCollectionPrefix": "%LEASE_COLLECTION_PREFIX%",
```

Tips: use `-` to separate prefix from the rest of the name.
```suggestion
      "leaseCollectionPrefix": "%LEASE_COLLECTION_PREFIX%-",
```